apps/web/src/common/components/ProfileImage/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+describe('<ProfileImage />', () => {
+  const defaultProps = {
+    src: 'https://picsum.photos/200',
+    userId: 1,
+    firstName: 'John',
+  }
+
+  const { default: ProfileImage } = require('.') as typeof import('.')
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('normal behavior', () => {
+    describe('display', () => {
+      it('should render ProfileImage correctly', () => {
+        render(<ProfileImage {...defaultProps} />)
+
+        // TODO: expect image to be rendered ater changed to next image
```

Comment:
(Just comment, please ignore this 😂 )
```suggestion
        // TODO: expect image to be rendered after changed to next image
```